src/MainContent.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import TaskForm from "./TaskForm";
+function MainContent() {
+  return (
+    <main>
```

Comment:
`<main>` will be probably shared by other parts in the app, as well as Footer and Header. Don't make it part of a particular component.

What you call `MainContent` is going to change in the future